tests/core/test_invite.py

```diff
@@ -36,18 +36,15 @@
 @pytest.mark.trio
 @pytest.mark.parametrize("with_labels", [False, True])
```

Comment:
```suggestion
@pytest.mark.parametrize("with_labels", [False, True], ids=["Without Labels", "With Labels"])
```
Can you apply that suggestion ? it's just to have a nicer `pytest` test trace

Reply:
👍 (and it would be better to avoid having a space in the label, this avoid having to use quotes when specifying the name of the test to run on the command line)

Reply:
Done